dist/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo List App</title>
+<script defer src="bundle.js"></script></head>
+<body>
```

Comment:
I see you're linking your JavaScript file in the dead tag. I too used to do that, but for good practices, it's best you link all your JS files at the bottom of the body tag just before the closing body tag. Doing that makes your index file looks very professional and followed good coding practices.

Reply:
@AmaduKamara  I didn't do that it's automatically generated  by webpack there

Reply:
Oh ok. No worries bro.